src/components/molecules/style.css

```diff
@@ -0,0 +1,4 @@
+.ball-holder {
+    display: flex;
+    flex-flow: row wrap;
+}
```

Comment:
Do you have your editor configured regarding line-endings?

Reply:
Yes, but a forgot to format again, already changed my configs to format on save now